src/content/docs/en/pages/main-menu/reference/build/edge-application/load-balancer.mdx

```diff
@@ -28,6 +28,12 @@ With Load Balancer, you can:
 
 The balancing method defines how the load will be distributed among your origins.
 
+:::note
+**Load Balancer** will query the origins according to the balancing method and weight they're assigned. If an origin returns a `4xx` or a `5xx` error, or takes longer to respond than the [timeouts](/en/documentation/products/edge-application/origins/#timeouts) defined in the origin, the Load Balancer will override the balancing method and weight to query the remaining origins before returning an error to your users.
```

Comment:
"Load Balancer"ou "the load balancer"?

Reply:
O produto mesmo, acho que não existe uma entidade load balancer, como é um origin type. :/